src/helpers/utils.ts

```diff
@@ -1,7 +1,27 @@
+import { camelCase } from 'lodash'
+
 export function sleep(secs: number) {
   return new Promise<void>((resolve) => {
     setTimeout(() => {
       resolve()
     }, secs * 1000)
   })
 }
+
+export function tabDelimitedToArray(text) {
```

Comment:
I strongly dislike that these are all in the same file

Reply:
Moved to export from index